build.sbt

```
@@ -1,25 +1,28 @@
 name := "sangria-play-json"
 organization := "org.sangria-graphql"
 version := "1.0.6-SNAPSHOT"
+mimaPreviousArtifacts := Set("org.sangria-graphql" %% "sangria-play-json" % "1.0.5")
 
 description := "Sangria play-json marshalling"
 homepage := Some(url("http://sangria-graphql.org"))
 licenses := Seq("Apache License, ASL Version 2.0" → url("http://www.apache.org/licenses/LICENSE-2.0"))
 
 scalaVersion := "2.12.7"
-crossScalaVersions := Seq("2.11.11", "2.12.7")
+crossScalaVersions := Seq("2.11.11", "2.12.7", "2.13.0")
```

We could use the `2.13` as default.
```suggestion
scalaVersion := "2.13.0"
crossScalaVersions := Seq("2.11.11", "2.12.7", scalaVersion.value)
```